crates/runtime/tests/models/mod.rs

```diff
@@ -164,3 +165,59 @@ fn normalize_search_response(mut json: Value) -> String {
 
     json.to_string()
 }
+
+fn normalize_embeddings_response(mut json: Value) -> String {
```

Comment:
This probably needs a docstring. If i understand what's happening, we're replacing the actual embedding vector, with its length?

Reply:
Documented - thank you!